README.adoc

```diff
@@ -314,9 +313,9 @@ the `FaultToleranceTest.java` file. Re-run the Maven build. You will see a test
 
 == Great work! You're done!
 
-You just built and tested a micorservice application with MicroProfile Fault Tolerance and Open Liberty.
+You built and tested a micorservice application with MicroProfile Fault Tolerance and Open Liberty.
```

Comment:
Other guide has, "You have just completed building and testing a microservice.." What do you think? And let's make it consistent.

Reply:
OK. We can make it match the other one.